ssl/ssl_test.cc

```diff
@@ -10071,6 +10525,150 @@ TEST(SSLTest, NameLists) {
   }
 }
 
+// Test a successful round-trip for KemKeyShare
+TEST(SSLTest, KemKeyShare) {
+  for (const GroupTest &t : kKemGroupTests) {
```

Comment:
If you want nicer test output and filtering you could make this a [parameterized test](https://github.com/google/googletest/blob/main/docs/advanced.md#value-parameterized-tests).

Reply:
Updated all new tests to be parameterized